lib/json_test.go

```diff
@@ -138,3 +138,24 @@ func TestCalcDiff(t *testing.T) {
 		assert.EqualValues(t, 400, stat[p.Prefix+".bar"])
 	}
 }
+
+func TestParseHeader(t *testing.T) {
+
+	host := "example.com"
+	userAgent := "mackerel-plugin-json/0.0.1"
+
+	h := []string{fmt.Sprintf("Host: %v", host)}
+
+	header, err := parseHeader(h)
+
+	assert.Nil(t, err)
+	assert.EqualValues(t, host, header.Get("Host"))
+
+	h = []string{fmt.Sprintf("Host: %v", host), fmt.Sprintf("User-Agent: %v", userAgent)}
+
+	header, err = parseHeader(h)
+
+	assert.Nil(t, err)
+	assert.EqualValues(t, host, header.Get("Host"))
+	assert.EqualValues(t, userAgent, header.Get("User-Agent"))
```

Comment:
We might change default `User-Agent` header in future, so testing with `User-Agent` header will conflict at that time.  testing with other headers will make more safe.

Reply:
I see. Tested with `Cache-Control` instead of `User-Agent`. https://github.com/mackerelio/mackerel-plugin-json/pull/5/commits/0435051f806e7bd82d73b0cdf4c6c1336d805a12